bin/build.sh

```diff
@@ -1,20 +1,39 @@
 #!/bin/bash
 set -ex
 
+# branch names don't correspond 1:1 with config file names
+branch="$1"
+case branch in
+    --dev)
+        SITE="dev"
+        ;;
+    --master)
+        SITE="stage"
+        ;;
+    --prod)
+        SITE="prod"
+        ;;
+    *)
+        SITE=""
+        ;;
+esac
+
 cd "$(dirname ${BASH_SOURCE[0]})"/..
 mkdir -p ./release
 
-# let gulp build the assets
-docker run --rm --label=gulp \
-    --volume=$(pwd):/srv \
-    --entrypoint /bin/bash \
-    huli/gulp \
-    -c "/entrypoint.sh build --production && chown $(id -u):$(id -g) . -R"
+if [[ -n $SITE ]]; then
```

Comment:
NIT: best to put double quotes around the variable like `"$SITE"` to ensure that it's interpreted as an empty string if it is, otherwise bash could consider it an error because nothing was passed to `-n`.

Reply:
Also, if you only want to build the site for these branches, you could instead just add this too the YAML for the build job:

```yaml
only:
  - dev
  - master
  - prod
```

But see my comment on the separation of jobs themselves on why we probably don't want this either.